src/components/section/ohri-form-section.scss

```diff
@@ -1,49 +1,77 @@
 @use '@carbon/colors';
 
-.section {
-  border-radius: solid 1px rgba(107, 104, 104, 0.5);
-  margin-bottom: 3.063rem;
+.parentResizer{
```

Comment:
Is this a cross-cutting styling or it's an obs group style? Plus for more consistency, let's properly format SCSS files:

```css
.myClassName {
```
👍 

```css
.myClassName{
```
👎